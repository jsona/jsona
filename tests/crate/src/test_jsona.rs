snapshot_jsona_syntax!("spec.jsona", syntax_spec);
snapshot_jsona_parse!("spec.jsona", parse_spec);
snapshot_jsona_format!("spec.jsona", format_spec);
