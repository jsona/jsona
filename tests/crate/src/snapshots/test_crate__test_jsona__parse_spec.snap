---
source: tests/crate/src/test_jsona.rs
expression: output
---
{
  "null": null,
  "boolTrue": true,
  "boolFalse": false,
  "float": 3.14,
  "floatNegative": -3.14,
  "floatNegativeWithoutInteger": -0.14,
  "floatNegativeWithoutDecimal": -3.0,
  "integer": 3,
  "integerNegative": -3,
  "integerHex": 26,
  "integerBinary": 1,
  "integerOctal": 10,
  "key \"single Quote\"": "value \"single Quote\"",
  "key 'double Quote'": "value 'double Quote'",
  "value 'backtick'\n\"quote\"": "value 'backtick'\n\"quote\"",
  "stringEscape1": "\u0000\b\f\n\r\t\u000b'\\©©你",
  "stringEscape2": "\u0000\b\f\n\r\t\u000b'\\©©你",
  "stringEscape3": "\u0000\b\f\n\r\t\u000b'\\©©你",
  "array": [
    "a",
    "b"
  ],
  "arrayEmpty": [],
  "arrayOnline": [
    "a",
    "b"
  ],
  "arrayExtraComma": [
    "a",
    "b"
  ],
  "object": {
    "k1": "v1",
    "k2": "v2"
  },
  "objectEmpty": {},
  "objectOneLine": {
    "k1": "v1",
    "k2": "v2"
  },
  "objectExtraComma": {
    "k1": "v1",
    "k2": "v2"
  }
}
